src/resolvers.ts

```diff
@@ -53,5 +53,40 @@ export const resolvers: ResolverMap = {
       console.warn(`Could not load value ${argument} from credstash: ${error}`);
       return undefined;
     });
+  },
+  aws: async (argument: string) => {
+    const params = argument.split('.');
```

Comment:
Can we enforce in some way that we only have two values in here?  Or support actual recursive lookup in your json code below?

Reply:
I've updated to use a recursive JSON approach. So you could nest your values as far as you want if you so choose.